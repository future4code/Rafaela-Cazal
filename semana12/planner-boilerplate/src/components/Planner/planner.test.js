import React from 'react';
import renderer from 'react-test-renderer';
import { Planner } from './index'

it('Snapshot component Planner', ()=> {
    const tree = renderer
    .create(<Planner></Planner>)
    .toJSON();
    expect(tree).toMatchSnapshot()
}) 