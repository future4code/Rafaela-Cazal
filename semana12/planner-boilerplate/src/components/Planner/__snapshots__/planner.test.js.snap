// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot component Planner 1`] = `
<div
  className="sc-bdVaJa gHLIV"
/>
`;
